tests/functional/controllers/test_restriction_controller.py

```diff
@@ -0,0 +1,309 @@
+from tensorhive.database import db_session
+from tensorhive.models.Restriction import Restriction
+from fixtures.controllers import API_URI as BASE_URI, HEADERS
+from http import HTTPStatus
+from sqlalchemy.orm.exc import NoResultFound
+
+import datetime
+import json
+import pytest
+
+ENDPOINT = BASE_URI + '/restrictions'
+
+
+# GET /restrictions
+def test_get_all_restrictions(tables, client):
```

Comment:
Shouldn't this test be divided into two separate ones?

Reply:
It could - done.